biz.aQute.bndlib/src/aQute/bnd/build/api/BuildInfo.java

```diff
@@ -0,0 +1,10 @@
+package aQute.bnd.build.api;
+
+import java.util.List;
+
+import aQute.bnd.build.Project;
+
+public interface BuildInfo {
+	Project getProject();
+	List<? extends ArtifactInfo> getArtifactInfos();
```

Comment:
This should be `List<ArtifactInfo>`. You never want to use wildcards in a return type since it messes up the code using your API.